projects/extension/src/background/index.ts

```diff
@@ -15,43 +10,12 @@ declare let window: Background
 
 const manager = (window.manager = new ConnectionManager())
 
-type RelayType = Map<string, string>
-
-export const relayChains: RelayType = new Map<string, string>([
-  ["polkadot", JSON.stringify(polkadot)],
-  ["kusama", JSON.stringify(kusama)],
-  ["rococo", JSON.stringify(rococo)],
-  ["westend", JSON.stringify(westend)],
-])
```

Comment:
IMO this should be left in the `index.ts`, and instead you add a new `trusted: bool` parameter to `addChain` indicating whether the chain is trusted.

Reply:
Can you please elaborate on the reasoning of having a `trusted: bool`. The idea behind this `Map` is that all of the chains are trusted. Maybe the `relayChains` naming should alter to `trustedChains` to be more obvious

Reply:
> In addition there is no reason for this to be in the index.ts

The reason is to leave the `ConnectionManager` as simple as possible. Adding these chains there just adds complexity to the code.

> Can you please elaborate on the reasoning of having a trusted: bool. The idea behind this Map is that all of the chains are trusted. Maybe the relayChains naming should alter to trustedChains to be more obvious.

This `trusted` parameter is way to move back the `relayChains` back to `index.ts`. If you leave `relayChains` in `ConnectionManager.ts` then indeed you don't need this parameter.